app/assets/javascripts/discourse/components/csv-uploader.js.es6

```diff
@@ -25,5 +25,30 @@ export default Em.Component.extend(UploadMixin, {
 
   uploadDone() {
     bootbox.alert(I18n.t("user.invited.bulk_invite.success"));
-  }
+  },
+
+  uploadOptions() {
+    return {
+      autoUpload: false,
+      confirmationMessage: I18n.t(
+        "user.invited.bulk_invite.confirmation_message"
+      )
+    };
+  },
+
+  _init: function() {
+    const $upload = this.$();
+
+    $upload.on("fileuploadadd", (e, data) => {
+      const confirmationMessage = I18n.t(
+        "user.invited.bulk_invite.confirmation_message"
+      );
+      bootbox.confirm(
+        confirmationMessage,
```

Comment:
You can replace the variable with the actual `I18n.t()` call

Reply:
haha, I was already working on fixing them after I saw your comment that I can move `fileuploadadd` part to CSV-uploader.
I didn't know I can override it inside CSV-uploader, new to this framework :slightly_smiling_face: 

Thanks for suggesting, this approach is much cleaner.